src/components/Results/Results.js

```diff
@@ -7,19 +7,36 @@ const Results = props => {
   // This is always `2`, as we have defined.
   const id = splittedPath[2];
 
-  const [data, setData] = useState({
-    responses: [],
-    id: "",
-    chennelID: "",
-    question: "",
-    channel: ""
-  });
+  const [data, setData] = useState({});
+  const responses = data.responses;
+
+  const nameFunction = () => {
+    let responseArray = [];
+    for (let response in responses) {
+      let vote = responses[response];
+      responseArray.push({ response, vote });
+      console.log("responseArray is", responseArray);
+    }
+    return responseArray;
+  };
```

Comment:
🏆 
Looks good , the only thing, did you want to name this function to something clearer?
And you can remove the console.log